document/document.go

```diff
@@ -30,9 +30,18 @@ func init() {
 }
 
 type Document struct {
-	id              string  `json:"id"`
-	Fields          []Field `json:"fields"`
-	CompositeFields []*CompositeField
+	id                string  `json:"id"`
+	Fields            []Field `json:"fields"`
+	CompositeFields   []*CompositeField
+	storedFieldsBytes uint64
+}
+
+func (d *Document) SetStoredFieldsBytes(val uint64) {
+	d.storedFieldsBytes = val
+}
+
+func (d *Document) GetStoredFieldsBytes() uint64 {
```

Comment:
This method can go away too, with storedFieldsBytes exported.

Reply:
This Get type of API might be necessary since the struct is hidden behind the Document interface.